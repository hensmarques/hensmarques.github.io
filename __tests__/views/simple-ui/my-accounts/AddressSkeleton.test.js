import React from 'react';
import AddressSkeleton from '../../../../src/views/simple-ui/my-accounts/AddressSkeleton';

describe('<AddressSkeleton/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<AddressSkeleton />);
    expect(wrapper).toMatchSnapshot();
  });
});
