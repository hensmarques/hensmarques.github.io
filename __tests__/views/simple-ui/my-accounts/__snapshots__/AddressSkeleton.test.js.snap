// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddressSkeleton/> renders and matches the snapshot 1`] = `
<Fragment>
  <div
    className="crypto-address-qr-code"
  >
    <div
      className="placeholder"
    />
  </div>
  <div
    className="crypto-address-text placeholder"
  >
    <div
      className="crypto-address-title placeholder"
    />
    <div
      className="crypto-address-code placeholder"
    />
  </div>
</Fragment>
`;
